api.planx.uk/modules/auth/middleware.ts

```diff
@@ -87,7 +87,14 @@ export const useFilePermission: RequestHandler = (req, _res, next): void => {
       req.headers["api-key"] as string,
       process.env.FILE_API_KEY_NEXUS!,
     ) ||
-    isEqual(req.headers["api-key"] as string, process.env.FILE_API_KEY_BARNET!);
+    isEqual(
+      req.headers["api-key"] as string,
+      process.env.FILE_API_KEY_BARNET!,
+    ) ||
+    isEqual(
+      req.headers["api-key"] as string,
+      process.env.FILE_API_KEY_SOUTHWARK!,
```

Comment:
Ideally we'd also constrain this to Southwark requests, but the current S3 setup doesn't support this.